packages/stream-tile/src/tile-document.ts

```diff
@@ -289,7 +274,7 @@ export class TileDocument<T = Record<string, any>> extends Stream {
     opts: UpdateOpts = {}
   ): Promise<void> {
     opts = { ...DEFAULT_UPDATE_OPTS, ...opts }
-    const signer: CeramicSigner = opts.asDID ? { did: opts.asDID } : this.api
```

Comment:
Or maybe it breaks compatibility here? The API seems to be different.

Reply:
Ah yeah, this would possibly break for someone, if they did opts that did something to `asDID`. Although that is mostly an internal method, someone could have a different implementation for it. I'm going to readd this method, then add a way to construct a signer from a did.